topics/statistics/tutorials/galaxy-ludwig/tutorial.md

```diff
@@ -0,0 +1,455 @@
+
+Notice that we are going to work with compressed file (.zip), Galaxy-Ludwig knows how to decompress the files and have them ready.
+
+![folder tree related to the hand-written digits picture](../../images/galaxy-ludwig/explain_images_file.png "File tree containing the images files used to training and testing the model")
+
+## MNIST_dataset.csv
+
+The MNIST dataset consists of images and their corresponding labels. For the purpose of this tutorial, mnist_dataset.csv file is created and contains three columns: image_path, label and, split.
+
+Briefly, the image_path column provides the file paths to the images that will be fed into the deep learning algorithm. The label column contains the correct classifications, ranging from 0 to 9, for the handwritten digits in the images. The split column indicates whether the data should be used for training (0) or testing (2) the model.
+
+![Dataset.csv file format snapshot](../../images/galaxy-ludwig/explain_dataset_format.png "Dataset.csv file format snapshot. features in order: file_path, label, split."){: width="50%" height="10%"}
+ 
+## Config.yaml
+
+The config.yaml file is crucial as it defines the entire structure of your machine learning experiment. This configuration file tells Ludwig how to process your data, what model to use, how to train it, and what outputs to generate.
+
+The rationale on how this file was constructed for this dataset is the following:
+i) The model takes images as input and uses a stacked convolutional neural network (CNN) to extract features.
+ii) It consists of two convolutional layers followed by a fully connected layer, with dropout applied to both the second convolutional layer and the fully connected layer to reduce overfitting. 
+iii) The split configuration is used to divide the dataset based on the 'split' column rather than using a random split (which is the default). In this configuration, rows labeled with '0' are included in the training set, while rows labeled with '2' are included in the testing set. If a validation category is needed, rows are labeled with '1'. 
+iv) The model is trained to classify images into categories (e.g., recognizing digits), and it will train for 5 epochs.
+
+![Config.yaml file snapshot](../../images/galaxy-ludwig/explain_config.png "Config.yaml file snapshot."){: width="50%" height="5%"}
+
+
+# Galaxy-Ludwig Tool
+
+Ludwig simplifies the complexities of machine learning by automating essential steps such as data preprocessing, model architecture selection, hyperparameter tuning, and device management. This streamlined approach is particularly beneficial for Galaxy users who are more interested in addressing their scientific questions than in navigating the intricacies of machine learning workflows.
```

Comment:
To automatically do `data preprocessing, model architecture selection, hyperparameter tuning, and device management`, what parameters should a user consider changing/updating based on a different dataset?

Reply:
I added some lines explaining and providing examples of modifications users can make to the configuration to better fit their dataset and produce a specific model.